File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.cs
public class Solution {
    public IList<bool> KidsWithCandies(int[] candies, int extraCandies) {
        int max = candies[0];
        foreach(int c in candies){
            if(c > max) max = c;
        }
        bool[] result = new bool[candies.Length];
        for(int i = 0; i < candies.Length; i++){
            result[i] = candies[i] + extraCandies >= max;
        }
        return result;
    }
}

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.js
var kidsWithCandies = function (candies, extraCandies) {
    let arr = [];
    let max = Math.max(...candies);
    for (c of candies) {
        (c + extraCandies >= max) ? arr.push(true) : arr.push(false);
    }
    return arr;
};